src/objective-c/tests/End2EndTest/End2EndTest/main.m

```diff
@@ -0,0 +1,16 @@
+//
+//  main.m
+//  End2EndTest
+//
+//  Created by mxyan on 7/11/16.
+//  Copyright © 2016 Google. All rights reserved.
+//
+
+#import <UIKit/UIKit.h>
+#import "AppDelegate.h"
+
+int main(int argc, char * argv[]) {
```

Comment:
It does not cos I removed main.m from the xcodeproj :) I should remove this from repository as well